src/routes/transactions/mappers/common/human-descriptions.mapper.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { encodeFunctionData, formatUnits, parseAbi } from 'viem';
+import { faker } from '@faker-js/faker';
+import { AddressInfo } from '../../../common/entities/address-info.entity';
+import { HumanDescriptionMapper } from './human-description.mapper';
+import { TokenRepository } from '../../../../domain/tokens/token.repository';
+import { tokenBuilder } from '../../../../domain/tokens/__tests__/token.builder';
+import { ILoggingService } from '../../../../logging/logging.interface';
+
+import { HumanDescriptionApi } from '../../../../datasources/human-description-api/human-description-api.service';
+import { MAX_UINT256 } from '../../constants';
+import { SafeAppInfo } from '../../entities/safe-app-info.entity';
+import { HumanDescriptionRepository } from '../../../../domain/human-description/human-description.repository';
+import { multisigTransactionBuilder } from '../../../../domain/safe/entities/__tests__/multisig-transaction.builder';
+import { SafeAppInfoMapper } from './safe-app-info.mapper';
+
+const tokenRepository = jest.mocked({
+  getToken: jest.fn(),
+} as unknown as TokenRepository);
+
+const mockLoggingService = {
+  info: jest.fn(),
+  debug: jest.fn(),
+  error: jest.fn(),
+  warn: jest.fn(),
+} as unknown as ILoggingService;
+
+const safeAppInfoMapper = jest.mocked({
+  mapSafeAppInfo: jest.fn(),
+} as unknown as SafeAppInfoMapper);
+
+const humanDescriptionAPI = new HumanDescriptionApi();
+const humanDescriptionRepository = new HumanDescriptionRepository(
+  humanDescriptionAPI,
+);
+const toAddress = new AddressInfo(faker.finance.ethereumAddress());
+const chainId = faker.string.numeric();
+const token = tokenBuilder()
+  .with('decimals', 18)
+  .with('name', 'Test Token')
+  .with('symbol', 'TST')
```

Comment:
We don't need to specifiy these for the builder as it generates them for us.

Reply:
I still have to limit the generated decimals otherwise `formatUnits` which uses `String.padStart` throws a range error if decimals is too big.